example/src/App.tsx

```diff
@@ -1,32 +1,52 @@
 import React, { useState } from 'react';
-import { useFinchConnect, SuccessEvent, ErrorEvent } from 'react-finch-connect';
+import { useFinchConnect, SuccessEvent, ErrorEvent } from '@tryfinch/react-connect';
+
 import './App.css';
 
 const App = () => {
-  const [code, setCode] = useState<string>();
+  const [sendState, setSendState] = useState<boolean>(false);
+  const [successEvent, setSuccessEvent] = useState<SuccessEvent>();
 
-  const onSuccess = ({ code }: SuccessEvent) => setCode(code);
+  const onSuccess = (e: SuccessEvent) => setSuccessEvent(e);
   const onError = ({ errorMessage }: ErrorEvent) => console.error(errorMessage);
   const onClose = () => console.log('User exited Finch Connect');
 
   const { open } = useFinchConnect({
-    clientId: '<your-client-id>',
+    clientId: 'c90b78c6-2151-4ca3-8fea-ccb708ffc5d9',
     products: ['company', 'directory', 'individual', 'employment'],
-    // sandbox: false,
+    sandbox: true,
     // payrollProvider: '<payroll-provider-id>',
     onSuccess,
     onError,
     onClose,
   });
 
+  const openFinchConnectClickHandler = () => open({
+    ...(sendState ? { state: new Date().toISOString() } : undefined),
+  });
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>Code: {code}</p>
-        <button type="button" onClick={() => open()}>
-          Open Finch Connect
-        </button>
-      </header>
+    <div className="container">
+      <h2><a href="https://www.npmjs.com/package/@tryfinch/react-connect">@tryfinch/react-connect</a> Example App</h2>
+      <form className="actions">
+        <div className="row">
+          <label className="top-label">Include State:</label>
+          <input type="checkbox" checked={sendState} onChange={() => setSendState(prev => !prev)} />
+        </div>
+        <div className="row">
+          <button className="cta" type="button" onClick={openFinchConnectClickHandler}>
```

Comment:
Make this a type of submit instead and move the callback to onSubmit on the form since the button is in a form
```suggestion
          <button className="cta" type="submit">
```